chatminer/chatparsers.py

```diff
@@ -237,12 +237,19 @@ def _read_file_into_list(self):
             self._logger.info(f"Finished reading {len(self.messages)} messages.")
 
     def _parse_message(self, mess):
-        parsed_message = {
-            "author": mess["from"],
-            "datetime": datetime.datetime.fromtimestamp(int(mess["date_unixtime"])),
-            "message": mess["text"],
-        }
-        return parsed_message
+        if (
+            "from" in mess
+            and "text" in mess
+            and type(mess["text"]) is str
+            and len(mess["text"]) > 0
```

Comment:
@galatolofederico Do we need these two additional checks? In which cases would they return `false`, while the first two checks return `true`?
If we don't need them, I would drop them for codestyle and performance reasons.